envoy_pkg/package_envoy.py

```diff
@@ -270,6 +295,8 @@ def main():
     else:
         if args.test_envoy:
             testEnvoy(args)
+        if args.upload and not args.override:
```

Comment:
This should be before `testEnvoy` which cost most time.

Reply:
Done.